src/Module/AudioTracksList.php

```diff
@@ -427,33 +425,32 @@ protected function parseItem(NewsModel $objItem, bool $blnAddArchive = false, st
         $objTemplate->timestamp = $objItem->date;
         $objTemplate->datetime = date('Y-m-d\TH:i:sP', (int) $objItem->date);
 
-        $imageFactory = System::getContainer()->get('contao.image.factory');
         // Retrieve and parse the picture
         if ($objItem->picture && $objFile = FilesModel::findByUuid($objItem->picture)) {
-            $objTemplate->picture =  $imageFactory->get($objFile->path, 300, 300);
+            $objTemplate->picture =  \Image::get($objFile->path, 300, 300);
```

Comment:
Would be cleaner to either : 
- add `use Contao\Image;`  & `$objTemplate->picture = Image::get`
- `$objTemplate->picture = \Contao\Image::get`
(I don't know what would happen if another vendor decides to also use `Image` as a class alias 🤔 )